src/frontend/Sider/Journalføring/Standard/JournalpostPanel.tsx

```diff
@@ -0,0 +1,196 @@
+
+import { FolderFileFillIcon, FolderFileIcon } from '@navikt/aksel-icons';
+import {
+    BodyShort,
+    Checkbox,
+    ExpansionCard,
+    Heading,
+    HStack,
+    Select,
+    VStack,
+} from '@navikt/ds-react';
+import { ABlue500 } from '@navikt/ds-tokens/dist/tokens';
+
+import { Journalføringsaksjon, JournalføringState } from '../../../hooks/useJournalføringState';
+import { Arkivtema, arkivtemaerTilTekst } from '../../../typer/arkivtema';
+import { Stønadstype, stønadstypeTilTekst } from '../../../typer/behandling/behandlingTema';
+import { Journalpost } from '../../../typer/journalpost';
+import { formaterIsoDato } from '../../../utils/dato';
+import { journalføringGjelderKlage } from '../Felles/utils';
+import { Journalføringsårsak, journalføringsårsakTilTekst } from '../typer/journalføringsårsak';
+
+const IkonContainer = styled.div`
+    color: ${ABlue500};
+`;
+
+const ExpansionCardHeader = styled(ExpansionCard.Header)`
+    padding-bottom: 0.25rem;
+`;
+
+const ExpansionCardContent = styled(VStack)`
+    padding-bottom: 1rem;
+`;
+
+const StyledSelect = styled(Select)`
+    max-width: 13rem;
+`;
+
+const StyledCheckbox = styled(Checkbox)`
+    margin-left: 1rem;
+`;
+
+const Grid = styled.div`
+    display: grid;
+    grid-template-columns: repeat(4, max-content);
+    column-gap: 1.5rem;
+`;
+
+interface Props {
+    journalpost: Journalpost;
+    journalpostState: JournalføringState;
+}
+
+const JournalpostPanel: React.FC<Props> = ({ journalpost, journalpostState }) => {
+    const {
+        journalføringsårsak,
+        settJournalføringsårsak,
+        stønadstype,
+        settStønadstype,
+        settJournalføringsaksjon,
+    } = journalpostState;
+    const [erPanelEkspandert, settErPanelEkspandert] = useState<boolean>(
+        journalføringsårsak === Journalføringsårsak.IKKE_VALGT || stønadstype === undefined
+    );
+
+    const tema = journalpost.tema
+        ? arkivtemaerTilTekst[journalpost.tema as Arkivtema]
+        : 'Tema ikke satt';
+    const datoMottatt = journalpostState.mottattDato
+        ? formaterIsoDato(journalpostState.mottattDato)
+        : 'Ikke satt';
+    const kanRedigere = journalføringsårsak !== Journalføringsårsak.DIGITAL_SØKNAD;
+    const klageGjelderTilbakekreving =
+        journalføringsårsak === Journalføringsårsak.KLAGE_TILBAKEKREVING;
+    const utledNesteJournalføringsårsak = (prevState: Journalføringsårsak) =>
+        prevState === Journalføringsårsak.KLAGE
+            ? Journalføringsårsak.KLAGE_TILBAKEKREVING
+            : Journalføringsårsak.KLAGE;
+    const valgbareStønadstyper = [Stønadstype.BARNETILSYN];
+    const valgbareJournalføringsårsaker = (årsak: Journalføringsårsak) => [
+        Journalføringsårsak.IKKE_VALGT,
+        Journalføringsårsak.ETTERSENDING,
+        årsak === Journalføringsårsak.KLAGE_TILBAKEKREVING
+            ? Journalføringsårsak.KLAGE_TILBAKEKREVING
+            : Journalføringsårsak.KLAGE,
+        Journalføringsårsak.PAPIRSØKNAD,
+    ];
```

Comment:
Her var det vanskelig å lese, trenger mer space. Tenker også at tema og dato mottatt kan bare gjøres der det brukes eller evt. flyttes nederst. `kanRedigere` bør lenger opp fordi det er viktigere osv. 

Kan noe av dette trekkes ut i en util? Feks valgbare ting?
```suggestion
    const kanRedigere = journalføringsårsak !== Journalføringsårsak.DIGITAL_SØKNAD;
    
const klageGjelderTilbakekreving =
    journalføringsårsak === Journalføringsårsak.KLAGE_TILBAKEKREVING;
    
const utledNesteJournalføringsårsak = (prevState: Journalføringsårsak) =>
    prevState === Journalføringsårsak.KLAGE
        ? Journalføringsårsak.KLAGE_TILBAKEKREVING
        : Journalføringsårsak.KLAGE;
        
const valgbareStønadstyper = [Stønadstype.BARNETILSYN];

const valgbareJournalføringsårsaker = (årsak: Journalføringsårsak) => [
    Journalføringsårsak.IKKE_VALGT,
    Journalføringsårsak.ETTERSENDING,
    årsak === Journalføringsårsak.KLAGE_TILBAKEKREVING
        ? Journalføringsårsak.KLAGE_TILBAKEKREVING
        : Journalføringsårsak.KLAGE,
    Journalføringsårsak.PAPIRSØKNAD,
];
```

Reply:
Trakk ut funksjoner til utils-filer så det ble litt mer oversiktelig: [87a0452](https://github.com/navikt/tilleggsstonader-sak-frontend/pull/261/commits/87a0452ff0a2f1912e690a7bb45507ce4d5f3a92)